pkg/apis/serving/v1alpha1/framework.go

```diff
@@ -39,6 +39,8 @@ const (
 var (
 	DefaultMemoryRequests = resource.MustParse("2Gi")
 	DefaultCPURequests    = resource.MustParse("1")
+	DefaultMemoryLimits   = resource.MustParse("2Gi")
```

Comment:
Should we just call this DefaultMemory and DefaultCPU and use the same variable for both?